detection_rules/kbwrap.py

```diff
@@ -178,20 +178,30 @@ def _process_imported_items(imported_items_list, item_type_description, item_key
 @click.option("--exceptions-directory", "-ed", required=False, type=Path, help="Directory to export exceptions to")
 @click.option("--default-author", "-da", type=str, required=False, help="Default author for rules missing one")
 @click.option("--rule-id", "-r", multiple=True, help="Optional Rule IDs to restrict export to")
+@click.option("--rule-name", "-rn", required=False, help="Optional Rule name to restrict export to (case-insensitive, supports wildcards)")
```

Comment:
```suggestion
@click.option("--rule-name", "-rn", required=False, help="Optional Rule name to restrict export to (KQL, case-insensitive, supports wildcards)")
```

Can we add a note like this to specifically inform the user that it is KQL? Thanks!